docs/source/traits_user_manual/defining.rst

```diff
@@ -480,8 +480,7 @@ value.
 
 A trait defined in this fashion can accept only values that are contained in
 the list of permitted values. The default value is the first value specified;
-it is also a valid value for assignment.
-::
```

Comment:
Hmm.  That's valid reST, and shouldn't prevent the syntax highlighting.  And indeed, if I restore the construct in this PR, the syntax highlighting remains.  I suspect that it's the changes below that actually make the difference to the highlighting.